### PR TITLE
feat: Forbid zero-amount void/refund/capture transactions [TA-1035]

### PR DESCRIPTION
**Pull request by szeying-lee**
# Description
* Removes `currency` from the example capture request
* Updates minimum value of capture request from `0` to `1`
* Adds explanation to void/refund/capture transaction endpoints regarding zero-amount transactions 

Fixes # (issue)
Re: **TA-1035**

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own changes
- [x] I have run `yarn test` to make sure my changes pass all linters
- [x] I have pulled the latest changes from the upstream `main` branch

## Contribution guidelines

For contribution guidelines, styleguide, and other helpful information please
see the `CONTRIBUTING.md` file in the root of this project.

### CODE REVIEW

File: reference/request-bodies/transaction_create_request.yaml [hunk 1]
```diff
@@ -17,6 +17,9 @@ properties:
       The monetary amount to create an authorization for, in the smallest
       currency unit for the given currency, for example `1299` cents to create
       an authorization for `$12.99`.
+
+      If the `intent` is set to `capture`, you must supply an amount that is
+      greater than zero.
```
Comment (hasier) [inline reference/request-bodies/transaction_create_request.yaml:22]: Maybe same here? 😬 

```If the `intent` is set to `capture`, an amount greater than zero must be supplied.```

**Review by hasier — APPROVED** [review]
approve

File: reference/request-bodies/transaction_create_request.yaml (hunk 1 above)
Comment (ipeluffo) [reply reference/request-bodies/transaction_create_request.yaml]: Done in https://github.com/gr4vy/gr4vy-openapi/pull/143/commits/a8c6b83340022740b9f2b8494f7439ba7fa2757a

**Review by hasier — DISMISSED** [review]

**Review by federicojasson — COMMENTED** [review]
Maybe this is a good time to change this text:

https://github.com/gr4vy/gr4vy-openapi/blob/a8c6b83340022740b9f2b8494f7439ba7fa2757a/reference/resources/transaction.yaml#L63
```
  captured_amount:
    type: integer
    description: |-
      The captured amount for this transaction. This can be a part and in some
      cases even more than the authorized amount.
    example: 999
    minimum: 0
    maximum: 99999999
```

**Pull request merged**